<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>433</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="recipeListWidget"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nameInput">
      <property name="placeholderText">
       <string>Recipe Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="categoryInput">
      <property name="placeholderText">
       <string>Category</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="ingredientsInput">
      <property name="placeholderText">
       <string>Ingredients (one per line)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="stepsInput">
      <property name="placeholderText">
       <string>Steps (one per line)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add Recipe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete Recipe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewButton">
      <property name="text">
       <string>View Recipe</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
